src/component/list.js

```diff
@@ -55,6 +55,9 @@ export class List extends Component {
         renderHeader={this.props.renderHeader}
         renderRow={this.props.renderItem}
         enableEmptySections={true}
+        horizontal={this.props.horizontal}
```

Comment:
Why do we want the list to scroll horizontally?

Reply:
True, I forget why i put that in. took out